vertexai/app/src/main/kotlin/com/google/firebase/quickstart/vertexai/GenerativeAiViewModelFactory.kt

```diff
@@ -106,6 +107,12 @@ val GenerativeViewModelFactory = object : ViewModelProvider.Factory {
                     AudioViewModel(generativeModel)
                 }
 
+                isAssignableFrom(ImagenViewModel::class.java) -> {
+                    val imagenModel = Firebase.vertexAI.imagenModel(
+                        "imagen-3.0-fast-generate-001")
```

Comment:
Shall we match the docs?

```suggestion
                        "imagen-3.0-generate-002")
```